Resources/public/js/views/services/plugins/ez-contentcreateplugin.js

```diff
@@ -46,10 +46,13 @@ YUI.add('ez-contentcreateplugin', function (Y) {
          * @param {eZ.ViewService} service
          */
         setNextViewServiceParameters: function (service) {
-            if ( this.get('contentType') && this.get('languageCode') && this.get('parentLocation') ) {
+            var app = this.get('host').get('app'),
+                defaultLanguageCode = app.get('defaultLanguageCode');
+
+            if ( this.get('contentType') && this.get('parentLocation') ) {
                 service.setAttrs({
                     contentType: this.get('contentType'),
-                    languageCode: this.get('languageCode'),
+                    languageCode: defaultLanguageCode,
```

Comment:
with this change, the `languageCode` attribute becomes useless even if it is filled with the language code coming from the `createContent` event or other said, the language code in `createContent` event is completely ignored.

That's not necessarily a bad thing (the whole `createContent` event is badly designed...) but I think you should add/do the following:

1. document the `createContent` event in https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/ez-contenttypeselectorview.js#L195 and mark the `languageCode` parameter as deprecated.
1. Remove that line https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/plugins/ez-contentcreateplugin.js#L121 to not use the deprecated event parameter
1. instead of changing the code here, you should just add a `valueFn` to the `languageCode` attribute to pick its default value from the app.